cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControlApp.java

```diff
@@ -4,7 +4,7 @@
 
 package com.linkedin.kafka.cruisecontrol;
 
-import com.codahale.metrics.JmxReporter;
+import com.codahale.metrics.jmx.JmxReporter;
```

Comment:
Let's avoid this change (please see earlier comment on the migration from `metrics-core 3.*` to `metrics-jmx 4.*`)